portfolio/src/main/webapp/index.html

```diff
@@ -9,10 +9,34 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Hello! My name is Eashan.</p>
+      <p>I'm a CS major at the Georgia Institute of Technology and I have a passion for social good. In an constantly technologizing society, there is only so much we can do to help others.
```

Comment:
are you able to fix the formatting so these lines aren't so long?

Reply:
Fixed